.github/workflows/deploy_development.yml

```diff
@@ -0,0 +1,28 @@
+name: Deploy to Firebase
+
+on:
+  push:
+    branches:
+      - '**'
```

Comment:
Not a criticism, I'm just wondering - what's the difference between ** and * here?

Reply:
`**` is the wildcard representation for all branch names, whereas `*` matches all branches without a `/`